libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -174,144 +182,135 @@ public interface VcFixtures {
+                            "24",
+                            "",
+                            "SOME STREET",
+                            "TE5 7ER",
+                            "SOME LOCALITY",
+                            "GB",
+                            TEST_UPRN,
+                            null,
+                            "2011-01-01"));
+
+    List<PostalAddress> MULTIPLE_ADDRESSES_NO_VALID_FROM =
             List.of(
                     ADDRESS_3,
-                    Map.ofEntries(
-                            Map.entry("uprn", "100120012077"),
-                            Map.entry("buildingNumber", "3"),
-                            Map.entry("buildingName", ""),
-                            Map.entry("streetName", "STOCKS HILL"),
-                            Map.entry("addressLocality", "HARRINGTON"),
-                            Map.entry("postalCode", "CA14 5PH"),
-                            Map.entry("addressCountry", "GB")),
-                    Map.ofEntries(
-                            Map.entry("uprn", "100120012077"),
-                            Map.entry("buildingNumber", "24"),
-                            Map.entry("buildingName", ""),
-                            Map.entry("streetName", "SOME STREET"),
-                            Map.entry("addressLocality", "SOME LOCALITY"),
-                            Map.entry("postalCode", "TE5 7ER"),
-                            Map.entry("addressCountry", "GB")));
-
-    Map<String, List<NameParts>> ALICE_PARKER_NAME =
+                    createPostalAddress(
+                            "3",
+                            "",
+                            "STOCKS HILL",
+                            "CA14 5PH",
+                            "HARRINGTON",
+                            "GB",
+                            TEST_UPRN,
+                            null,
+                            null),
+                    createPostalAddress(
+                            "24",
+                            "",
+                            "SOME STREET",
+                            "TE5 7ER",
+                            "SOME LOCALITY",
+                            "GB",
+                            TEST_UPRN,
+                            null,
+                            null));
+
+    Map<String, List<NamePart>> ALICE_PARKER_NAME =
             Map.of(
                     VC_NAME_PARTS,
                     List.of(
-                            new NameParts("Alice", VC_GIVEN_NAME),
-                            new NameParts("Jane", VC_GIVEN_NAME),
-                            new NameParts("Parker", VC_FAMILY_NAME)));
-    Map<String, List<NameParts>> MORGAN_SARAH_MEREDYTH_NAME =
+                            createNamePart("Alice", NamePart.NamePartType.GIVEN_NAME),
+                            createNamePart("Jane", NamePart.NamePartType.GIVEN_NAME),
+                            createNamePart("Parker", NamePart.NamePartType.FAMILY_NAME)));
```

Comment:
(this is fine, but I'd also be happy for these to be static imports - i.e. so it's `createNamePart("Alice", GIVEN_NAME)`)